UnityProject/Assets/Scripts/Managers/MatrixManager/CentralCommandBeacon.cs

```diff
@@ -0,0 +1,17 @@
+using System.Collections;
+using System.Collections.Generic;
+using Managers;
+using UnityEngine;
+
+public class CentralCommandBeacon : MonoBehaviour
```

Comment:
where's the namespace?

Reply:
what should be the namespace?

Reply:
look at the file structure and go off from there.
here it would be `Managers.MatrixManager`